docs/targets/solana.rst

```diff
@@ -73,6 +73,36 @@ package has `documentation <https://solana-labs.github.io/solana-solidity.js/>`_
 are `some examples <https://solana-labs.github.io/solana-solidity.js/>`_. There is also
 `solang's integration tests <https://github.com/hyperledger/solang/tree/main/integration/solana>`_.
 
+.. _call_anchor:
+
+Calling Anchor Programs from Solidity
+_____________________________________
+
+It is possible to call `Anchor Programs <https://github.com/coral-xyz/anchor>`_
+from Solidity. You first have to generate interface file from the IDL file using
+the :ref:`metadata_command`. Then, import the Solidity file in your Solidity using the
+``import "...";`` syntax. Say you have an anchor program called `remoge` with a
+function `nerp`, you can call it like so:
```

Comment:
How did you come up with `remoge` and `nerp`? 😂

Reply:
I used a random word generator. Suggestions welcome!

Reply:
How about renaming `remoge` to `solidity_from_idl` and `nerp` to `idl_defined_function`?
I tend to name my examples with self-explanatory names so that they become even easier to understand. What do you think?

Reply:
fixed